### PR TITLE
MAINT: dropping python3.8 support

### PR DESCRIPTION
**Pull request by bsipocz**
I have some issues locally when running the tests, please don't merge this just yet even if CI is all green, so I can ensure that it's indeed just a local issue (I don't see it with tox, but do see it when running tests directly in the source)

### CODE REVIEW

File: tests/test_doctestplus.py [hunk 1]
```diff
@@ -314,7 +314,7 @@ def test_normalize_with_ellipsis(self):
         c = OutputChecker()
         got = []
         for char in ['A', 'B', 'C', 'D', 'E']:
-            got.append('%s %s' % (char, float(ord(char))))
+            got.append('{} {}'.format(char, float(ord(char))))
```
Comment (pllim) [inline tests/test_doctestplus.py:317]: Hmm why no f-string here?

Comment (bsipocz) [reply tests/test_doctestplus.py:317]: Automated mess I guess 🤷‍♀️

I mean, I pushed stuff before boarding a flight, but as I said above there is some weird stuff happening locally already so I'll very likely have to follow up with more manual fixes

Comment (pllim) [reply tests/test_doctestplus.py:317]: No rush on my end. Thanks!

Comment (bsipocz) [reply tests/test_doctestplus.py:317]: local issues were due to outdate plugins

**Pull request merged**